devpack/src/main/java/io/neow3j/devpack/contracts/NeoToken.java

```diff
@@ -48,10 +49,20 @@ public class NeoToken extends FungibleToken {
     public static native boolean vote(Hash160 scriptHash, ECPoint candidatePubKey);
 
     /**
-     * @return the list of registered candidates.
+     * @return the list of the current validators.
```

Comment:
More precisely this returns the first 256 registered candidates. Again I don't agree with "validators" here.

Reply:
You're right, I'll change it.